<!DOCTYPE html>
<!-- Generated by pkgdown: do not edit by hand --><html lang="en">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<title>Comparative Analysis with Annotation-Based Genomic Components • CALANGO</title>
<!-- favicons --><link rel="icon" type="image/png" sizes="16x16" href="favicon-16x16.png">
<link rel="icon" type="image/png" sizes="32x32" href="favicon-32x32.png">
<link rel="apple-touch-icon" type="image/png" sizes="180x180" href="apple-touch-icon.png">
<link rel="apple-touch-icon" type="image/png" sizes="120x120" href="apple-touch-icon-120x120.png">
<link rel="apple-touch-icon" type="image/png" sizes="76x76" href="apple-touch-icon-76x76.png">
<link rel="apple-touch-icon" type="image/png" sizes="60x60" href="apple-touch-icon-60x60.png">
<!-- jquery --><script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.4.1/jquery.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script><!-- Bootstrap --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/css/bootstrap.min.css" integrity="sha256-bZLfwXAP04zRMK2BjiO8iu9pf4FbLqX6zitd+tIvLhE=" crossorigin="anonymous">
<script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.4.1/js/bootstrap.min.js" integrity="sha256-nuL8/2cJ5NDSSwnKD8VqreErSWHtnEP9E7AySL+1ev4=" crossorigin="anonymous"></script><!-- bootstrap-toc --><link rel="stylesheet" href="bootstrap-toc.css">
<script src="bootstrap-toc.js"></script><!-- Font Awesome icons --><link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/all.min.css" integrity="sha256-mmgLkCYLUQbXn0B1SRqzHar6dCnv9oZFPEC1g1cwlkk=" crossorigin="anonymous">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.12.1/css/v4-shims.min.css" integrity="sha256-wZjR52fzng1pJHwx4aV2AO3yyTOXrcDW7jBpJtTwVxw=" crossorigin="anonymous">
<!-- clipboard.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/clipboard.js/2.0.6/clipboard.min.js" integrity="sha256-inc5kl9MA1hkeYUt+EC3BhlIgyp/2jDIyBLS6k3UxPI=" crossorigin="anonymous"></script><!-- headroom.js --><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/headroom.min.js" integrity="sha256-AsUX4SJE1+yuDu5+mAVzJbuYNPHj/WroHuZ8Ir/CkE0=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/headroom/0.11.0/jQuery.headroom.min.js" integrity="sha256-ZX/yNShbjqsohH1k95liqY9Gd8uOiE1S4vZc+9KQ1K4=" crossorigin="anonymous"></script><!-- pkgdown --><link href="pkgdown.css" rel="stylesheet">
<script src="pkgdown.js"></script><meta property="og:title" content="Comparative Analysis with Annotation-Based Genomic Components">
<meta property="og:description" content="A first-principle, phylogeny-aware comparative genomics tool for 
             investigating associations between terms used to annotate genomic
             components (e.g., Pfam IDs, Gene Ontology terms,) with quantitative 
             or rank variables such as number of cell types, genome size, or 
             density of specific genomic elements. See the project website for 
             more information, documentation and examples.">
<meta property="og:image" content="/logo.svg">
<!-- mathjax --><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js" integrity="sha256-nvJJv9wWKEm88qvoQl9ekL2J+k/RWIsaSScxxlsrv8k=" crossorigin="anonymous"></script><script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/config/TeX-AMS-MML_HTMLorMML.js" integrity="sha256-84DKXVJXs0/F8OTMzX4UR909+jtl4G7SPypPavF+GfA=" crossorigin="anonymous"></script><!--[if lt IE 9]>
<script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
<script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
<![endif]-->
</head>
<body data-spy="scroll" data-target="#toc">
    <div class="container template-home">
      <header><div class="navbar navbar-default navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar" aria-expanded="false">
        <span class="sr-only">Toggle navigation</span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <span class="navbar-brand">
        <a class="navbar-link" href="index.html">CALANGO</a>
        <span class="version label label-default" data-toggle="tooltip" data-placement="bottom" title="Released version">1.0.8</span>
      </span>
    </div>

    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
<li>
  <a href="index.html">
    <span class="fas fa-home fa-lg"></span>
     
  </a>
</li>
<li>
  <a href="reference/index.html">Reference</a>
</li>
<li class="dropdown">
  <a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-expanded="false">
    Articles
     
    <span class="caret"></span>
  </a>
  <ul class="dropdown-menu" role="menu">
<li>
      <a href="articles/CALANGO_Parameters.html">CALANGO Parameters</a>
    </li>
    <li>
      <a href="articles/examples-page.html">CALANGO Examples</a>
    </li>
  </ul>
</li>
<li>
  <a href="news/index.html">Changelog</a>
</li>
      </ul>
<ul class="nav navbar-nav navbar-right">
<li>
  <a href="https://github.com/fcampelo/CALANGO/">
    <span class="fab fa-github fa-lg"></span>
     
  </a>
</li>
      </ul>
</div>
<!--/.nav-collapse -->
  </div>
<!--/.container -->
</div>
<!--/.navbar -->

      

      </header><div class="row">
  <div class="contents col-md-9">
<div id="calango-comparative-analysis-with-annotation-based-genomic-componentes" class="section level1">
<div class="page-header"><h1 class="hasAnchor">
<a href="#calango-comparative-analysis-with-annotation-based-genomic-componentes" class="anchor"></a>CALANGO<br><font size="+2">Comparative AnaLysis with ANnotation-based Genomic cOmponentes</font>
</h1></div>
<!-- badges: start -->

<p><img src="https://github.com/fcampelo/CALANGO/raw/master/inst/images/CALANGO_LOGO-lighter.svg" height="300" alt="CALANGO logo. Drawn by Brazilian artist Berze - https://www.facebook.com/berzearte"></p>
<hr>
<div id="description" class="section level3">
<h3 class="hasAnchor">
<a href="#description" class="anchor"></a>DESCRIPTION</h3>
<p>CALANGO is a first-principles, phylogeny-aware comparative genomics package to search for annotation terms (e.g Pfam IDs, GO terms or superfamilies), formally described in a dictionary-like structure and used to annotate genomic components, associated with a quantitative/rank variable (e.g. number of cell types, genome size or density of specific genomic elements).</p>
<p>Our software has been freely inspired by (and explicitly modeled to take into account information from) ideas and tools as diverse as comparative phylogenetics, genome annotation, gene enrichment analysis, and data visualization and interactivity.</p>
</div>
<div id="how-to-install" class="section level3">
<h3 class="hasAnchor">
<a href="#how-to-install" class="anchor"></a>HOW TO INSTALL</h3>
<p>The latest version of CALANGO can be installed directly from the repository using <code><a href="https://devtools.r-lib.org/reference/remote-reexports.html">devtools::install_github()</a></code>:</p>
<pre><code><a href="https://devtools.r-lib.org/reference/remote-reexports.html">devtools::install_github("fcampelo/CALANGO")</a></code></pre>
<p>Alternatively, you should soon be able to install the last release version from CRAN by simply doing:</p>
<pre><code><a href="https://rdrr.io/r/utils/install.packages.html">install.packages("CALANGO")</a></code></pre>
<p>In either case, please make sure that you have the latest R version (at least 3.6.1) as well as updated versions of all installed packages.</p>
</div>
<div id="installing-and-updating-dependencies" class="section level3">
<h3 class="hasAnchor">
<a href="#installing-and-updating-dependencies" class="anchor"></a>INSTALLING AND UPDATING DEPENDENCIES</h3>
<p>CALANGO depends on some packages from <strong>Bioconductor</strong>. This requires an extra installation step before CALANGO can be used. Just run:</p>
<pre><code><a href="https://fcampelo.github.io/CALANGO/">library(CALANGO)
install_bioc_dependencies(which = "all")</a></code></pre>
<p>to install / update all dependencies (packages listed as CALANGO’s <code>imports</code> and <code>suggests</code>) to their latest versions.</p>
<hr>
</div>
<div id="how-to-use---overview" class="section level3">
<h3 class="hasAnchor">
<a href="#how-to-use---overview" class="anchor"></a>HOW TO USE - OVERVIEW</h3>
<p>To run CALANGO you need two things:</p>
<ol>
<li>a set of data<br>
</li>
<li>an input list defining the path to that data and what is to be done.</li>
</ol>
<p><strong>Retrieving data</strong></p>
<p>A set of example data folders and files can accessed directly from the package using function <code><a href="reference/retrieve_data_files.html">retrieve_data_files()</a></code>. For instance, a call:</p>
<div class="sourceCode" id="cb4"><pre class="downlit sourceCode r">
<code class="sourceCode R"><span class="fu">CALANGO</span><span class="fu">::</span><span class="fu"><a href="reference/retrieve_data_files.html">retrieve_data_files</a></span><span class="op">(</span><span class="st">"./data"</span><span class="op">)</span></code></pre></div>
<p>will create a folder called <code>data</code> in the current working directory path, and download the sample files into subfolders within it. These subfolders will contain:</p>
<ul>
<li>genome annotation data files (under <code>./data/Pfam/</code> or <code>./data/gene2GO/</code>)</li>
<li>metadata files (under <code>./data/metadata/</code>)</li>
<li>phylogenetic tree files (under <code>./data/trees/</code>)</li>
<li>dictionary files (under <code>./data/dics/</code>)</li>
</ul>
<p>The downloaded data will also contain files that describe the input list mentioned above, under <code>./data/parameters/</code>. These full examples represent all input files required to locally reproduce several types of analyses.</p>
<hr>
<p>Once these two pieces are in place, the CALANGO pipeline can be run by invoking the main function of the package. For instance:</p>
<pre><code>output &lt;- run_CALANGO(defs = "./data/parameters/parameters_domain2GO_freq.txt", cores = 2)</code></pre>
<p>This call will generate an enriched <code>CALANGO</code> list as the output, and generate a dynamic HTML5 output (plus several tab-separated value (tsv) files in the directory provided as <code>output.dir</code>.</p>
</div>
<div id="preparing-your-input-files" class="section level3">
<h3 class="hasAnchor">
<a href="#preparing-your-input-files" class="anchor"></a>PREPARING YOUR INPUT FILES</h3>
<p>CALANGO requires the following files (please check the examples in in <code>./data/parameters/</code> if in doubt about file specifications):</p>
<hr>
<p><strong>genome annotation file</strong></p>
<p>A text file for each species describing their set of biologically meaningful genomic elements and their respective annotation IDs (e.g. non redundant proteomes annotated to GO terms, or non-redundant protein domains annotated to protein domain IDs). An example of such file, where gene products are annotated using Gene Ontology (GO) terms and Kegg Orthology (KO) identifiers would be as follows:</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode sh"><code class="sourceCode bash"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a><span class="ex">Entry</span>   GO_IDs   KEGG_Orthology_ID</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a><span class="ex">Q7L8J4</span>  GO:0017124<span class="kw">;</span><span class="ex">GO:0005737</span><span class="kw">;</span><span class="ex">GO:0035556</span><span class="kw">;</span><span class="ex">GO:1904030</span><span class="kw">;</span><span class="ex">GO:0061099</span><span class="kw">;</span><span class="ex">GO:0004860</span></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a><span class="ex">Q8WW27</span>  GO:0016814<span class="kw">;</span><span class="ex">GO:0006397</span><span class="kw">;</span><span class="ex">GO:0008270</span>  K18773</span>
<span id="cb6-4"><a href="#cb6-4" aria-hidden="true" tabindex="-1"></a><span class="ex">Q96P50</span>  GO:0005096<span class="kw">;</span><span class="ex">GO:0046872</span>   K12489</span></code></pre></div>
<p>And is specified as:</p>
<ul>
<li><p>Fixed number of columns per file (minimum 2) separated by tabs, or “\t”.</p></li>
<li><p>First line as the header, each column having a unique column name.</p></li>
<li><p>Each following line having an instance of a genomic feature represented by an unique ID (a specific coding-gene locus found in a genome, for instance) identifier.</p></li>
<li><p>Multiple terms of the same entry (e.g. a gene annotated to multiple GO terms) should be in the same column and separated by “;”, with any number of spaces before and after it. It’s use after the last term is optional.</p></li>
<li><p>A column can have no terms in any given row.</p></li>
</ul>
<p>In a more abstract representation, files representing genome annotations for a single annotation schema would have two columns and the following general structure:</p>
<div class="sourceCode" id="cb7"><pre class="sourceCode sh"><code class="sourceCode bash"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="ex">genomic_element_name/ID_1</span>     annotation_ID_1<span class="kw">;(</span><span class="ex">...</span><span class="kw">);</span><span class="ex">annotation_ID_N</span></span>
<span id="cb7-2"><a href="#cb7-2" aria-hidden="true" tabindex="-1"></a><span class="ex">genomic_element_name/ID_2</span>     annotation_ID_12</span></code></pre></div>
<hr>
<p><strong>phylogenetic tree file</strong></p>
<p><em>newick</em> or <em>PHYLIP</em> format, containing at least:</p>
<ul>
<li><p>all species to be analyzed (species IDs in the tree must be the same name of text files from the previous step)</p></li>
<li><p>branch lengths proportional to divergence times (a chronogram)</p></li>
<li><p>no polytomies (if there are such cases, CALANGO will resolve star branches using <a href="https://rdrr.io/cran/ape/man/multi2di.html">multi2di</a> method as implemented in <a href="http://ape-package.ird.fr/">ape</a> package.</p></li>
</ul>
<p>A tree in <em>newick</em> format (however, with no branch lengths), would be:</p>
<div class="sourceCode" id="cb8"><pre class="sourceCode sh"><code class="sourceCode bash"><span id="cb8-1"><a href="#cb8-1" aria-hidden="true" tabindex="-1"></a><span class="kw">(</span><span class="ex">genome_ID_1,</span><span class="er">(</span><span class="ex">genome_ID_2,genome_ID_3</span><span class="kw">))</span></span></code></pre></div>
<hr>
<p><strong>A metadata file</strong></p>
<p>Containing species-specific information:</p>
<ul>
<li><p>genome ID column (same name of text files and of species in phylogenetic trees, must be the first column);</p></li>
<li><p>The quantitative/rank variable used to sort/rank genomes;</p></li>
<li><p>A normalizing factor (e.g. proteome size or length; number of annotation terms) to correct for potential biases in datasets (e.g. organisms with different annotation levels or with highly discrepant proteome sizes) If users are using GO as annotation schema, CALANGO can compute a normalizing factor that takes into account all GO counts, including internal nodes, and therefore may not provide any normalizing factor for this case.</p></li>
</ul>
<p>The tabular format for the correlation analysis where column 1 contains the genome IDs, column 2 contains the variable to rank genomes and column 3 contains the normalizing factor could be as follows:</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode sh"><code class="sourceCode bash"><span id="cb9-1"><a href="#cb9-1" aria-hidden="true" tabindex="-1"></a><span class="ex">../projects/my_project/genome_ID_1</span>  1.7  2537</span>
<span id="cb9-2"><a href="#cb9-2" aria-hidden="true" tabindex="-1"></a><span class="ex">../projects/my_project/genome_ID_2</span>  1.2  10212</span>
<span id="cb9-3"><a href="#cb9-3" aria-hidden="true" tabindex="-1"></a><span class="ex">../projects/my_project/genome_ID_3</span>  0.9  1534</span></code></pre></div>
<p>Metadada files are specified as follows:</p>
<ul>
<li><p>Fixed number of columns (minimum 2) separated by tabs.</p></li>
<li><p>No header.</p></li>
<li><p>Each line having a unique identifier (first column) with the path of the genome it is referring to or to genome ID. Referred as “genome ID column”.</p></li>
<li><p>One mandatory numeric value in every other column, referred as “variable columns”.</p></li>
</ul>
<hr>
<p><strong>A dictionary</strong></p>
<p>Tab delimited, linking annotation IDs to their biologically meaningful descriptions. Our software currently supports two dictionary types:</p>
<ul>
<li><p>Gene Ontology (GO) - in this case, CALANGO will automatically recover annotation description and compute values for internal GOs not explicitly used to describe data annotation.</p></li>
<li><p>other - in this case, users need to describe a new dictionary linking annotation term IDs (e.g. “46456”) to their descriptions (e.g. “All alpha proteins”), separated by tabs. An example of such file would be:</p></li>
</ul>
<div class="sourceCode" id="cb10"><pre class="sourceCode sh"><code class="sourceCode bash"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="ex">Annotation_ID</span>     Annotation_definition</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a><span class="ex">annotation_ID_1</span>   All alpha proteins</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a><span class="ex">annotation_ID_2</span>   Globin-like</span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a><span class="ex">annotation_ID_3</span>   Globin-like</span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a><span class="ex">annotation_ID_4</span>   Truncated hemoglobin</span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a><span class="kw">(</span><span class="ex">...</span><span class="kw">)</span></span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a><span class="ex">annotation_ID_N</span>   annotation_ID_description</span></code></pre></div>
<p>CALANGO can treat each identifier as its own description, preventing the need to prepare an ontology that isn’t natively supported. For that, do not specify any ontology file and set the <code>ontology = "other"</code>.</p>
</div>
<div id="setting-up-calango-parameters" class="section level3">
<h3 class="hasAnchor">
<a href="#setting-up-calango-parameters" class="anchor"></a>SETTING UP CALANGO PARAMETERS</h3>
<p>CALANGO’s parameters are listed in the documentation of <code><a href="reference/run_CALANGO.html">run_CALANGO()</a></code>, as well as in the examples file provided (<code>./data/parameters/</code>). They are, for the current version:</p>
<ul>
<li><p>annotation.files.dir (required, string) - Folder where annotation files are located.</p></li>
<li><p>output.dir (required, string) - output folder for results</p></li>
<li>
<p>dataset.info (required, string) - genome metadata file, it should contain at least:</p>
<ol>
<li>path for annotation data (if “annotation.files.dir” not provided OR file names (if "annotation.files.dir) is provided. Please notice this information should be the first column in metadata file;</li>
<li>phenotype data (numeric, this is the value CALANGO uses to rank species when searching for associations)</li>
<li>normalization data (numeric, this is the value CALANGO uses as a denominator to compute annotation term frequencies to remove potential biases caused by, for instance, over annotation of model organisms or large differences in the counts of genomic elements). Please notice CALANGO does not require normalization data for GO, as it computes the total number of GO terms per species and uses it as a normalizing factor.</li>
</ol>
</li>
<li><p>x.column (required, numeric) - which column in “dataset.info” contains the phenotype data?</p></li>
<li><p>ontology (required, string) - which dictionary data type to use? Possible values are “GO” and “other”. For GO, CALANGO can compute normalization data.</p></li>
<li><p>dict.path (required, string) - file for dictionary file (two-column file containing annotation IDs and their descriptions, not needed for GO</p></li>
<li><p>column (required, string) - which column in annotation files should be used (column name)</p></li>
<li><p>denominator.column (optional, numeric) - which column contains normalization data?</p></li>
<li><p>tree.path (required, string) - path for tree file in either <em>newick</em> or <em>nexus</em> format</p></li>
<li><p>tree.type (required, string) - tree file type (either “nexus” or “newick”, case-sensitive)</p></li>
<li><p>cores (optional, numeric) - how many cores to use? If not provided the function defaults to 1.</p></li>
<li><p>linear.model.cutoff = 0.5 (required, numeric) - Parameter that regulates how much graphical output is produced. We configure it to generate plots only for annotation terms with corrected q-values for phylogenetically independent contrasts smaller than 0.5.</p></li>
<li><p>MHT.method = “BH” (optional, string) - type of multiple hypothesis correction to be used. Accepts all methods listed by <code><a href="https://rdrr.io/r/stats/p.adjust.html">stats::p.adjust.methods()</a></code>. If not provided the function defaults to “BH”.</p></li>
<li><p>type = “correlation” (optional, string) type of analysis to perform. Currently only “correlation” is supported.</p></li>
</ul>
</div>
<div id="calango-output" class="section level3">
<h3 class="hasAnchor">
<a href="#calango-output" class="anchor"></a>CALANGO OUTPUT</h3>
<p>Live examples of CALANGO output HTML5 pages can be found <a href="https://fcampelo.github.io/CALANGO/articles/examples-page.html" target="_blank">here</a>.</p>
<p>CALANGO produces as its main output a dynamic HMTL5 website containing:</p>
<ul>
<li>A description of the input values and a button to download the input and results list.</li>
<li>Interactive heatmaps and scatterplots</li>
<li>A dynamic table of results.</li>
</ul>
<p>Please check our examples page at <a href="https://fcampelo.github.io/CALANGO/" class="uri">https://fcampelo.github.io/CALANGO/</a> to explore the full output of CALANGO for a variety of examples. The required data to fully reproduce these examples can be obtained by using <code><a href="reference/retrieve_data_files.html">CALANGO::retrieve_data_files()</a></code>.</p>
<hr>
</div>
<div id="calango-mastermind" class="section level3">
<h3 class="hasAnchor">
<a href="#calango-mastermind" class="anchor"></a>CALANGO MASTERMIND</h3>
<ul>
<li>Francisco P. Lobo (<a href="mailto:franciscolobo@gmail.com" class="email">franciscolobo@gmail.com</a>, <a href="mailto:francisco.lobo@ufmg.br" class="email">francisco.lobo@ufmg.br</a>)
<ul>
<li>Main developer: initial idea, scientific coordination, first implementation, main visualization concepts.</li>
</ul>
</li>
</ul>
</div>
<div id="other-developers" class="section level3">
<h3 class="hasAnchor">
<a href="#other-developers" class="anchor"></a>OTHER DEVELOPERS</h3>
<ul>
<li>Felipe Campelo (<a href="mailto:f.campelo@aston.ac.uk" class="email">f.campelo@aston.ac.uk</a>, <a href="mailto:fcampelo@gmail.com" class="email">fcampelo@gmail.com</a>)
<ul>
<li>Code auditing and refactoring to R package, CRAN deployment, visualization support.<br>
</li>
</ul>
</li>
<li>Giovanni M. de Castro (<a href="mailto:giomcastro@gmail.com" class="email">giomcastro@gmail.com</a>)
<ul>
<li>Initial development and implementation support.<br>
</li>
</ul>
</li>
<li>Jorge A. Hongo (<a href="mailto:jorgeahongo@gmail.com" class="email">jorgeahongo@gmail.com</a>)
<ul>
<li>Initial development and implementation support.</li>
</ul>
</li>
</ul>
</div>
<div id="non-coding-collaborators" class="section level3">
<h3 class="hasAnchor">
<a href="#non-coding-collaborators" class="anchor"></a>NON-CODING COLLABORATORS</h3>
<ul>
<li>Gabriel M. F. Almeida - scientific adviser (analysis of phage data)</li>
</ul>
<hr>
</div>
</div>
  </div>

  <div class="col-md-3 hidden-xs hidden-sm" id="pkgdown-sidebar">
    <div class="links">
<h2>Links</h2>
<ul class="list-unstyled">
<li>Download from CRAN at <br><a href="https://cloud.r-project.org/package=CALANGO">https://​cloud.r-project.org/​package=CALANGO</a>
</li>
<li>Browse source code at <br><a href="https://github.com/fcampelo/CALANGO/">https://​github.com/​fcampelo/​CALANGO/​</a>
</li>
<li>Report a bug at <br><a href="https://github.com/fcampelo/CALANGO/issues/">https://​github.com/​fcampelo/​CALANGO/​issues/​</a>
</li>
</ul>
</div>
<div class="license">
<h2>License</h2>
<ul class="list-unstyled">
<li><a href="https://www.r-project.org/Licenses/GPL-2">GPL-2</a></li>
</ul>
</div>
<div class="developers">
<h2>Developers</h2>
<ul class="list-unstyled">
<li>Francisco Lobo <br><small class="roles"> Author </small>  </li>
<li>Felipe Campelo <br><small class="roles"> Author, maintainer </small>  </li>
<li>Jorge Augusto Hongo <br><small class="roles"> Author </small>  </li>
<li>Giovanni Marques de Castro <br><small class="roles"> Author </small>  </li>
<li><a href="authors.html">All authors...</a></li>
</ul>
</div>

  <div class="dev-status">
<h2>Dev status</h2>
<ul class="list-unstyled">
<li><a href="https://github.com/fcampelo/CALANGO/actions"><img src="https://github.com/fcampelo/CALANGO/workflows/R-CMD-check/badge.svg" alt="R-CMD-check"></a></li>
<li><a href="https://CRAN.R-project.org/package=CALANGO"><img src="https://www.r-pkg.org/badges/version/CALANGO" alt="CRAN_Status_Badge"></a></li>
<li><a href="https://CRAN.R-project.org/package=CALANGO"><img src="https://cranlogs.r-pkg.org/badges/CALANGO" alt="CRAN Downloads"></a></li>
</ul>
</div>
</div>
</div>


      <footer><div class="copyright">
  <p>Developed by Francisco Lobo, Felipe Campelo, Jorge Augusto Hongo, Giovanni Marques de Castro.</p>
</div>

<div class="pkgdown">
  <p>Site built with <a href="https://pkgdown.r-lib.org/">pkgdown</a> 1.6.1.</p>
</div>

      </footer>
</div>

  


  </body>
</html>
